modules/cnf-tuning-nodes-for-low-latency-via-performanceprofile.adoc

```diff
@@ -55,10 +55,78 @@ spec:
 [id="cnf-partitioning-the-cpus_{context}"]
 == Partitioning the CPUs
 
-You can reserve cores, or threads, for operating system housekeeping tasks from a single NUMA node and put your workloads on another NUMA node. The reason for this is that the housekeeping processes might be using the CPUs in a way that would impact latency sensitive processes running on those same CPUs. Keeping your workloads on a separate NUMA node prevents the processes from interfering with each other. Additionally, each NUMA node has its own memory bus that is not shared.
+// Module included in the following assemblies:
```

Comment:
I'm a little confused by what this topic is doing here. You could simply include the topic instead of copying and pasting and having to do dual maintenance.